.ci/Jenkinsfile

```diff
@@ -224,7 +224,7 @@ pipeline {
         axes {
           axis {
             name 'K8S_VERSION'
-            values "v1.24.0", "v1.23.6", "v1.22.9", "v1.21.12"
+            values "v1.25.0","v1.24.3", "v1.23.9", "v1.22.12"
```

Comment:
Are we sure we want to remove testing for `v1.21` already?

Reply:
Well it's already EoL https://endoflife.date/kubernetes, let's reduce the list to imrpove CI time